satellite/gracefulexit/endpoint.go

```diff
@@ -267,6 +267,7 @@ func (endpoint *Endpoint) doProcess(stream processStream) (err error) {
 		return nil
 	}
 
+	// maps pieceIDs to pendingTransfer
```

Comment:
nit - I think something like `maps pieceIDs to pendingTransfers to keep track of ongoing piece transfer requests` could be more useful for providing the reader with context.